MVCapplication/src/main/java/com/softserve/if072/mvcapp/controller/StorePageController.java

```diff
@@ -171,4 +171,44 @@ public String deleteStore(@RequestParam("storeId") int storeId) {
         }
     }
 
+    @GetMapping("/editStore")
+    public String editStore(@RequestParam("storeId") String storeId, ModelMap model) {
+        final String uri = storeUrl + "/{storeId}";
+        RestTemplate restTemplate = getRestTemplate();
+        idStore = Integer.parseInt(storeId);
+
+        try {
+            Map<String, Integer> param = new HashMap<>();
+            param.put("storeId", Integer.parseInt(storeId));
+            Store store = restTemplate.getForObject(uri, Store.class, param);
+            model.addAttribute("store", store);
+            LOGGER.info("Editing Store         " + idStore);
+            return "editStore";
+
+        } catch (Exception e) {
+            LOGGER.error(String.format("Store with id %d is not possible to edit", storeId));
+            return "redirect:/stores/";
+        }
+    }
+
+    @PostMapping("/editStore")
+    public String editStore(@ModelAttribute("store") Store store) {
+        store.setId(idStore);
```

Comment:
As I understand, you need to have <form:hidden> tag in the jsp with storeId, in this case you will avoid having storeId as class field in the controller